management/baremetal.md

```diff
@@ -0,0 +1,58 @@
+##Preparing baremetal or personal VM environments for Cluster Management
+
+This document goes through the steps to prepare a baremetal or personal VM setup for cluster management.
+
+**Note:**
+- Unless explicitly mentioned all the steps below are done by logging into the same host. It is referred as *control host* below.
+- Right now we test cluster manager on Centos7.2. More OS variations shall be added in future.
+
```

Comment:
@mapuri Can i try this out on a bare metal node running RHEL 7.2 stock os? Or this is in pipeline for next drop?